CHANGELOG.rst

```diff
@@ -13,6 +13,8 @@ adheres to `Semantic Versioning <http://semver.org/spec/v2.0.0.html>`_.
 
 Added
 ~~~~~
+- Precision parameter for stft
```

Comment:
Maybe a tad more explanations would be nice, e.g., (float32 vs float64). Does not need to be long.